SteamReview/SteamReview/Steam.cpp

```diff
@@ -110,11 +119,24 @@ void FSteam::OpenCategoryManager()
 void FSteam::OpenDisplayGames()
 {
 	std::system("cls");
-	Uncategorized.DisplayCategory();
+	FGame UncategorizedGame;
```

Comment:
This is such a strange behaviour, and it has to do with the way you changed your DisplayCategory function, you now have to create a Game that is never used, only to be able to call the DisplayCategory method.